site/docs/getting-started/roadmap.mdx

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: Roadmap
+hide_table_of_contents: true
+---
+
+import Content from "@site/src/layouts/content/Content.tsx";
+
+# Roadmap
+
+<Content>
+
+## Q1 2023 delivery
+
+This is the collaborative estimate of the design system team for realistic delivery within Q1.
+
+It is intended as a guide to our intended released work, for guidance and information. It should not be considered an absolute commitment.
+
+### System
+
+#### Components
+
+- [Tooltip](https://github.com/jpmorganchase/salt-ds/issues/324)
```

Comment:
If these are ordered by priority, perhaps it would be better to use a numbered list to indicate that.

Otherwise (if we don't want to convey a priority order within the quarter), I would suggest sorting them alphabetically by component name, so that it's easier to find a given component name.

Reply:
Numbered lists should be reserved for sequential steps, alphabetically makes sense.

Reply:
I agree, it should be alphabetical.

Reply:
It's a good question. It's currently mirroring the component order here - https://github.com/orgs/jpmorganchase/projects/5/views/54. So it is roughly intended to convey prioritization order. But that prioritization order can change.

I will chat to @joshwooding about making our project board (or parts of it) public. Then we can put alphabetically here, and also link to the GitHub section that has more specific priority ordering.

Reply:
I wouldn't change it to alphabetical since it's been ordered to represent priority. Let me know when you want to chat @shinytoyrobots

Reply:
We know it's priority order, but there's nothing on the page to make that apparent to readers, so it just looks like a jumbled list.

If alphabetical order or using a numbered list aren't options, then I suggest we at least add an introductory sentence along the lines of "Component work has been prioritized into the following order" (I'll let someone else come up with better wording, but hopefully you get the jist of what I'm trying to convey).